### ISSUE TITLE
Topic /motion/cmd_vc already has a callback registered

### ISSUE THREAD

**Issue by Hoolo** [Fontys-GTL/AlphaCeresMotion#1]
When running this code as a docker container, it crashes after connecting to the odrives. This (for now) only seems to happen on the RPI in the robot and using the hardware odrives.

### How to replicate

- Open 2 terminals on the Pi.
- If the image has not been built yet, build it with `invoke build-image`
- In terminal 1 start the mqtt broker: go to the integration folder and run `docker compose up`
- In terminal 2 start the docker container. `docker run -it --network="host" local/alpha-motion bash` We have to use --network="host" to get access to the physical odrives (and probably mqtt).
- Set the correct variables for our hardware setup in terminal 2: `export machine_wheel_ids=[0,1]` and `export machine_can_channel=slcan0`
- Then simply start the machine via `motion run machine`. This should trigger the error as shown below.

### Possible solutions
Either mqtt is not set up correctly (wrong order of declaring callback & connecting?), or some kind of issue with the docker image.

### Error stacktrace
![image](https://github.com/user-attachments/assets/9a5781c0-b9f6-4a16-af15-20d1feea48bf)

**Comment by Hoolo** [comment]
Running this sequence works fine when this repository is opened as a devcontainer instead of building the image (running with mock drives & motion run machine from a terminal in the devcontainer).

When building the image on my laptop and using docker to start a container from this image, it does not work. Running with mock drives it gives the exact same error message as on the pi, so it seems like an issue with the built image.

**Comment by sjev** [comment]
I've fixed this on `fix-issue-1` branch. 

The fix was just remove:
`self._coros.append(self._on_init)`   from `machine.py`.

However, there was a danging branch with quite some changes, which I integrated in this one. 
After testing, this should be merged into main branch. 

**Note** setting can inteface is now done with 

    "CAN_CHANNEL": "vcan0",
    "CAN_INTERFACE": "socketcan"

environment variables
